analyzer/evmtokenbalances/evm_token_balances.go

```diff
@@ -286,3 +291,7 @@ func (m Main) Start(ctx context.Context) {
 func (m Main) Name() string {
 	return EvmTokenBalancesAnalyzerPrefix + string(m.runtime)
 }
+
+func nativeTokenSymbol(sdkPT *sdkConfig.ParaTime) string {
+	return sdkPT.Denominations[sdkConfig.NativeDenominationKey].Symbol
```

Comment:
did we really not have this anywhere?

Reply:
We have it once in test code, and once somewhere else but it's a method of a struct that contains `sdkConfig.ParaTime` :)

Reply:
> it's a method of a struct that contains `sdkConfig.ParaTime` :)

oh no that was me! :see_no_evil: